Engine/Code/Engine/Graphics/Renderer.h

```diff
@@ -89,7 +89,7 @@ class Renderer
 	void DrawQuad(const Vec2& position, const Vec2& dimensions, const Texture& texture, const AABB2& texCoords, const Vec4& color, Shader shader);
 	void DrawQuad(const Vec2& position, Vec2 meshDim, Vec4 color, const std::string& path, Shader shader);
 
-	void DrawMeshImmediate(const Vec2& position, Vec2 meshDim);
+	void DrawFullScreenQuad(const Vec2& position = Vec2(0.0f, 0.0f), Vec2 meshDim = Vec2(1024.0f, 1024.0f));
```

Comment:
You shouldn't need these params in the first place. We can change this after this merge.